parser/openfoodfacts_taxonomy_parser/parser.py

```diff
@@ -199,7 +215,9 @@ def harvest(self, filename):
         index_stopwords = 0
         index_synonyms = 0
         language_code_prefix = re.compile("[a-zA-Z][a-zA-Z][a-zA-Z]?:")
-        self.stopwords = dict()
+        self.stopwords = (
+            dict()
+        )  # it will contain a list of stopwords with their language code as key
```

Comment:
why not simply put the comment before the line ? That's the way we do normally !

Also better use the `{}` to init a dict (not a big deal though)

```suggestion
        # stopwords will contain a list of stopwords with their language code as key
        self.stopwords = {}
```